src/main/java/pl/sknikod/kodemy/util/BaseCrudOperations.java

```diff
@@ -0,0 +1,24 @@
+package pl.sknikod.kodemy.util;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.List;
+
+public interface BaseCrudOperations<REQ, RES> {
+    @GetMapping
+    public ResponseEntity<List<RES>> findAll();
+
+    @GetMapping("/{id}")
```

Comment:
Zastanawiam się czy nie lepiej byłoby z `get`. W stylu:
- `getAll()`
- `getById()`.
Ogólnie to chcemy dostać, a nie znaleźć.